nmtwizard/preprocess/tokenizer.py

```diff
@@ -63,3 +63,15 @@ def make_subword_learner(subword_config, subword_dir, tokenizer=None):
         "subword_type": subword_type,
         "size": vocab_size
     }
+
+def load_vocabulary(vocabulary_path):
+    if vocabulary_path and isinstance(vocabulary_path, str):
+        vocabulary = []
+        with open(vocabulary_path, "rt") as vocabfile:
+            for line in vocabfile:
+                if line.startswith('#'):
```

Comment:
"#" might be a token. So this condition should not apply past the header. See how https://github.com/OpenNMT/nmt-wizard-docker/blob/master/nmtwizard/framework.py#L942 is implemented with the `header` variable.